setup/blocklist.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Add Blocklist.de malicious IP Addresses to Daily Crontab
+# Also IPtables-persistent to save IP addresses upon reboot
+# Added by Alon "ChiefGyk" Ganon
+# alonganon.info
+# alon@ganon.me
+cp conf/blocklist/sync-fail2ban /etc/cron.daily/sync-fail2ban
+chmod a+x /etc/cron.daily/sync-fail2ban
+time /etc/cron.daily/sync-fail2ban
+echo iptables-persistent iptables-persistent/autosave_v4 boolean true | sudo debconf-set-selections
```

Comment:
I pulled sync-fail2ban from a curl I mentioned in the original issue #864 and added the IPTables persistance, I will have to remove root I basically copied it from a script I created separately and overlooked it. So I will have to remove that. The script actually will run independent of fail2ban as I have it as it's own thing, and have tested it to run on a VPS without Fail2ban.